### PR TITLE
add changelog

### PR DESCRIPTION
**Pull request by evgeni**

### CODE REVIEW

**Comment by evgeni** [comment]
another one for the #910/#911 todolist

**Comment by evgeni** [comment]
and I swear, I totally didn't implement #916 to have it used in the very next PR ;)

**Comment by evgeni** [comment]
`No module named rstcheck` -- seems we have missing dependencies for sanity

**Comment by evgeni** [comment]
:green_apple:

**Comment by mdellweg** [comment]
Is the CHANGELOG.rst generated from the yaml version?

**Comment by evgeni** [comment]
> Is the CHANGELOG.rst generated from the yaml version?

Yes it is. (by running `antsibull-changelog generate`)

**Comment by mdellweg** [comment]
> > Is the CHANGELOG.rst generated from the yaml version?
> 
> Yes it is. (by running `antsibull-changelog generate`)

It seemed too redundant to require me typing it twice. ;)
Do you plan to document the process of generating the Changelog? Or will this be automated in the release Workflow?

**Comment by evgeni** [comment]
>  Do you plan to document the process of generating the Changelog? Or will this be automated in the release Workflow?

Just pushed an update to `releasing.md`. Don't get confused that I use `a-c release` there, this does more than just the generation from the yaml (the yaml needs to be generated from the fragments first)

File: requirements-dev.txt [hunk 1]
```diff
@@ -8,4 +8,5 @@ rpm-py-installer
 docker
 -r requirements.txt
 -r https://github.com/ansible/ansible/raw/devel/test/lib/ansible_test/_data/requirements/sanity.pylint.txt
+-r https://github.com/ansible/ansible/raw/devel/test/lib/ansible_test/_data/requirements/sanity.rstcheck.txt
 -r https://github.com/ansible/ansible/raw/devel/test/lib/ansible_test/_data/requirements/sanity.validate-modules.txt
```
Comment (mdellweg) [inline requirements-dev.txt]: Are you sure we still need those?
I don't see them in https://github.com/pulp/squeezer anymore.

Comment (evgeni) [reply requirements-dev.txt]: the validate-modules ones? maybe not. but didn't dare to try.

**Review by ehelms — APPROVED** [review]

**Review by mdellweg — APPROVED** [review]

**Pull request merged**